### ISSUE TITLE
[BITV] 9.2.4.3/3.4 - Navigation on the left side / both desktop and mobile views - When the navigation is collapsed, it is still possible to navigate through the hidden elements using the keyboard. This should be prevented. (8)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36990]
- [x] Fix
- [x] Server and Photos have to be updated to NC Vue v7.11.0

![5e2cbbfe0fd4258eab94ad65371fb1fa](https://user-images.githubusercontent.com/1315170/222248375-3aac61af-5bfa-422d-87c8-b16daf84ccfa.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-89806fe784-v3-n4

</details>

- [ ] 9.2.4.3/4.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-89806fe784-v4-n2
- [ ] 9.2.4.3/5.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-89806fe784-v5-n2
- [ ] 9.2.4.3/6.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-89806fe784-v6-n2
- [ ] 9.2.4.3/7.3 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-89806fe784-v7-n3
- [ ] 9.2.4.3/8.3 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-89806fe784-v8-n3
- [ ] 9.2.4.3/9.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-89806fe784-v9-n2
- [ ] 9.2.4.3/10.4 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-89806fe784-v10-n4
